src/main/java/org/opensearch/ad/settings/ADEnabledSetting.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.ad.settings;
```

Comment:
Should we delete the current `EnabledSettings.java` file later?

Reply:
same goes for numericsettings and other copied over files

Reply:
yes, just deleted EnabledSetting and NumericSetting